.travis.yml

```diff
@@ -0,0 +1,16 @@
+before_script:
+  - wget https://github.com/openshift/source-to-image/releases/download/v1.1.10/source-to-image-v1.1.10-27f0729d-linux-amd64.tar.gz
```

Comment:
There is a 1.1.13 release of `s2i`. Should this maybe use the latest?

Reply:
Ah, I did not know that. I just took this as is from the centos-s2i-image repo.